src/sentry/utils/snuba.py

```diff
@@ -904,9 +906,8 @@ def _raw_snql_query(
             )
             body = query.snuba()
 
-        with thread_hub.start_span(op="snuba_snql", description="run query") as span:
+        with thread_hub.start_span(op="snuba_snql.run", description=str(query)) as span:
```

Comment:
database spans also have the query in the description. is the problem with span search that the span data is gone or that the description is too high-card?

Reply:
That the description is too high-card, which means we can't use the feature to find anything interesting using the span search feature

Also the database spans are normalized which works decently with span search, but these snql queries aren't if I recall correctly.

Reply:
>Also the database spans are normalized which works decently with span search, but these snql queries aren't if I recall correctly.

We do this because of PII concerns and SDK performance _specifically in Python_, and there's customers taking issue with that too. I believe the search feature will have to adapt then, there's already too much data out in the wild that looks like that.

Reply:
Isn't PII an issue here? This will potentially expose our customer's data in our transactions.

Reply:
i'm not adding or removing data here at all, and PII in SDKs is a completely different situation than in concrete deployments